Formula/hepmc3.rb

```diff
@@ -1,10 +1,10 @@
 class Hepmc3 < Formula
   desc "C++ event record for Monte Carlo generators"
   homepage "https://hepmc.web.cern.ch/"
-  url "https://hepmc.web.cern.ch/hepmc/releases/HepMC3-3.2.5.tar.gz"
-  sha256 "cd0f75c80f75549c59cc2a829ece7601c77de97cb2a5ab75790cac8e1d585032"
-  license "GPL-3.0-or-later"
-  revision 2
+  url "https://gitlab.cern.ch/hepmc/HepMC3/-/archive/3.2.6/HepMC3-3.2.6.tar.gz"
+  sha256 "648a3725a772e3dc26eaba0fe3c2f941a38d2b39679cea64413a2ae49ad946b0"
+  license "LGPL-3.0-or-later"
+  revision 1
```

Comment:
This line should be removed altogether (revision is only needed when a formula changes but the version remains the same).
```suggestion
```

Reply:
Done.